### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by nghiapq77** [izderadicka/pdfparser#25]
```
import pdfparser.poppler as pdf
doc = pdf.Document(b'40.pdf')
for page in doc:
    for flow in page:
        for block in flow:
            for line in block:
                print(line.text)
```
[40.pdf](https://github.com/izderadicka/pdfparser/files/3503950/40.pdf)

And I got Segmentation fault (core dumped)
Any help would be appreciated, thank you

Edit 1: I figured this has something to do with the font

**Comment by izderadicka** [comment]
Please provider your platform, python version,  libpoppler version, is this document only one causing prpblem?

**Comment by nghiapq77** [comment]
> Please provider your platform, python version, libpoppler version, is this document only one causing prpblem?

So far it's the only one. Im using Ubuntu 18.04, python 3.6.9 and libpoppler 0.62.0

**Comment by izderadicka** [comment]
Looks like it's regression from latest commit c6357b2 ,  previous commit seems to works on this file. 
Latest version has segfault problem on given file - just after `15. SHIPPING MARK` header. The text below has strange font, so could be related to some fonts issue as suggested.

Will need to look into in in more detail.  

Can you please in meanwhile check if previous commit works for you too?

@DainisGorbunovs  - can you please check if above referred file fails in your build too, thanks.

**Comment by izderadicka** [comment]
Ok so problem was when libpoppler returns NULL as font name - in  latest change it was not handled properly, thus in those cases we got segfault. I've tried to fix in fa04b07 - confirm please that it does solve your issue.

**Comment by DainisGorbunovs** [comment]
Thanks for reporting this issue, and thanks for tagging me.

I was looking into it, and can confirm that the regression is due to [`c6357b2`](https://github.com/izderadicka/pdfparser/commit/c6357b2093ded1d82e512523ab9436423db11a4e).

@izderadicka: your commit [`fa04b07`](https://github.com/izderadicka/pdfparser/commit/fa04b078d9a757e178aab9908602c391a2e02570) successfully fixes the issue, and am able to parse `40.pdf`.
